### DISCUSSION TITLE
Unloaded RealESRGAN. 😨

### DISCUSSION THREAD

**Discussion by ZeroCool22** [hlky/stable-diffusion-webui#798] [Q&A] [answered]
![sdsssaaaaaaa](https://user-images.githubusercontent.com/13344308/189001532-3a4aa5c9-e1a3-4c86-a067-71d1a7555379.png)

I put it in the correct folder:

![Screenshot_4](https://user-images.githubusercontent.com/13344308/189001604-f9b561f9-20e7-4998-a9a7-8a92e5a4bb51.jpg)

It's normal?

**Comment by EliEron** [comment, marked as answer]
Yes, it's completely normal. 

Having Real-ESRGAN and GFPGAN loaded consumes VRAM, so to reduce VRAM load only one of them is currently loaded at once. 

So when you use Real-ESRGAN the GFPGAN model is automatically unloaded, and when using GFPGAN the Real-ESRGAN model is automatically unloaded.
